### PR TITLE
Improve ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by php-coder**
I hope that this extra paragraph help our users to submit their issues into a proper repository. Any feedback is welcome!

PTAL @pweil- @sdodson @openshift/team-documentation 
CC @simo5

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<!--
+Note that this is the issue tracker for OpenShift Origin.
+
+OpenShift Installer issues should be reported at https://github.com/openshift/openshift-ansible/issues
```
Comment (mfojtik) [inline .github/ISSUE_TEMPLATE.md]: maybe do similar for web-console? cc @jwforres

Comment (jwforres) [reply .github/ISSUE_TEMPLATE.md]: Yes please. Console issues get filed in the wrong place a lot.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+Documentation issues is better to report at https://github.com/openshift/openshift-docs/issues
```
Comment (mburke5678) [inline .github/ISSUE_TEMPLATE.md]: What is this page for? I wonder if there is a better mechanism for users to report documentation issues, if this page is for end users. Not sure how closely the openshift-docs/issues is monitored.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (php-coder) [reply .github/ISSUE_TEMPLATE.md]: @jwforres Added. PTAL.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (php-coder) [reply .github/ISSUE_TEMPLATE.md]: @mburke5678 This is a template for a new issues. See the following link for details:
- https://help.github.com/articles/creating-an-issue-template-for-your-repository/
- https://github.com/blog/2111-issue-and-pull-request-templates

> I wonder if there is a better mechanism for users to report documentation issues, if this page is for end users.

I don't know about such mechanism. Time to time, users reporting documentation issues here, but it happens not every day, so if you object against this paragraph, I will remove it.

**Comment by sdodson** [comment]
/approve
Fine with me

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (mburke5678) [reply .github/ISSUE_TEMPLATE.md]: @php-coder Please leave the paragraph in. That is great!

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+<!--
+Note that this is the issue tracker for OpenShift Origin.
+
+OpenShift Installer issues should be reported at https://github.com/openshift/openshift-ansible/issues
+Management Console issues are being collected at https://github.com/openshift/origin-web-console
```
Comment (jwforres) [inline .github/ISSUE_TEMPLATE.md]: link should go to the issues page

Comment (php-coder) [reply .github/ISSUE_TEMPLATE.md]: Agree. Updated. PTAL.

**Comment by jwforres** [comment]
Console bit looks fine
/approve

**Comment by php-coder** [comment]
Test flake #16273
/test cmd

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+<!--
+Note that this is the issue tracker for OpenShift Origin.
+
+OpenShift Installer issues should be reported at https://github.com/openshift/openshift-ansible/issues
+Management Console issues are collected at https://github.com/openshift/origin-web-console/issues
+Documentation issues are better reported at https://github.com/openshift/openshift-docs/issues
```
Comment (adellape) [inline .github/ISSUE_TEMPLATE.md]: :+1:

**Comment by php-coder** [comment]
@soltysh Could you look at it and put LGTM if it deserves? :)

**Comment by soltysh** [comment]
I see my buddy @pweil- was faster than I ;)

**Pull request merged**